### PR TITLE
change admin user to be named operator

### PR DESCRIPTION
**Pull request by MiaAltieri**
This follows the format of the other DPE repos

### CODE REVIEW

**Review by WRFitch — APPROVED** [review]

File: lib/charms/tls_certificates_interface/v1/tls_certificates.py [hunk 1]
```diff
@@ -126,6 +126,7 @@ def _on_certificate_revocation_request(self, event: CertificateRevocationRequest
 from charms.tls_certificates_interface.v1.tls_certificates import (
```
[the remark this answers is not in the document]
Comment (MiaAltieri) [reply lib/charms/tls_certificates_interface/v1/tls_certificates.py:126]: yes!

File: actions.yaml [hunk 2]
```diff
@@ -7,11 +7,20 @@ get-primary:
 get-password:
   description: Change the admin user's password, which is used by charm.
     It is for internal charm users and SHOULD NOT be used by applications.
+  params:
+    username:
+      type: string
+      description: The username, the default value 'operator'.
+        Possible values - operator, pbmuser.
```
Comment (delgod) [inline actions.yaml:14]: lets rename the user to `backup`
```suggestion
        Possible values - operator, backup.
```

File: lib/charms/mongodb/v0/mongodb.py [hunk 3]
```diff
@@ -35,7 +35,7 @@
 logger = logging.getLogger(__name__)
 
 # List of system usernames needed for correct work on the charm.
-CHARM_USERS = ["operator"]
+CHARM_USERS = ["operator", "pbm"]
```
Comment (delgod) [inline lib/charms/mongodb/v0/mongodb.py:38]: lets rename the user to `backup`
```suggestion
CHARM_USERS = ["operator", "backup"]
```

**Review by delgod — CHANGES_REQUESTED** [review]

File: actions.yaml [hunk 4]
```diff
@@ -7,11 +7,20 @@ get-primary:
...
+        Possible values - operator, backup.
 
 set-password:
   description: Change the admin user's password, which is used by charm.
     It is for internal charm users and SHOULD NOT be used by applications.
   params:
+    username:
+      type: string
+      description: The username, the default value 'operator'.
+        Possible values - operator, pbmuser.
```
Comment (delgod) [inline actions.yaml:23]:
```suggestion
        Possible values - operator, backup.
```

**Review by delgod — APPROVED** [review]

**Pull request merged**